deployment/keystone/changeset/workflowregistry/update_allowed_dons.go

```diff
@@ -0,0 +1,51 @@
+package workflowregistry
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+
+	kslib "github.com/smartcontractkit/chainlink/deployment/keystone"
+	workflow_registry "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/workflow/generated/workflow_registry_wrapper"
+)
+
+var _ deployment.ChangeSet[*UpdateAllowedDonsRequest] = UpdateAllowedDons
+
+type UpdateAllowedDonsRequest struct {
```

Comment:
this will need to support mcms

you can do it now or in a followup, but can't be deferred for long [eg](https://github.com/smartcontractkit/chainlink/blob/fd9d5de76f4d86555472c21fb184ebaa82b143b7/deployment/keystone/changeset/update_don.go#L24)

Reply:
I think MCMS should be supported here.